src/axom/quest/examples/quest_candidates_example.cpp

```diff
@@ -0,0 +1,754 @@
+/// spatial index, either a Bounding Volume Hierarchy or an Implicit Grid.
+/// The example supports HIP and CUDA execution through RAJA.
+//-----------------------------------------------------------------------------
+
+#include "axom/config.hpp"
+#include "axom/core.hpp"
+#include "axom/slic.hpp"
+
+#ifdef AXOM_USE_RAJA
+  #include "RAJA/RAJA.hpp"
+#else
+  #error This example requires axom to be configured with RAJA support
+#endif
+
+#ifdef AXOM_USE_UMPIRE
+  #include "umpire/Umpire.hpp"
+#else
+  #error This example requires axom to be configured with Umpire support
+#endif
+
+#ifdef AXOM_USE_CONDUIT
+  #include "conduit_relay.hpp"
+  #include "conduit_blueprint.hpp"
+#else
+  #error This example requires axom to be configured with Conduit support
+#endif
+
+#include "axom/mint.hpp"
+#include "axom/primal.hpp"
+#include "axom/spin.hpp"
+#include "axom/quest.hpp"
+
+#include "axom/fmt.hpp"
+#include "axom/CLI11.hpp"
+
+#include <memory>
+#include <array>
+
+using seq_exec = axom::SEQ_EXEC;
+
+using UMesh = axom::mint::UnstructuredMesh<axom::mint::SINGLE_SHAPE>;
+using IndexPair = std::pair<axom::IndexType, axom::IndexType>;
+
+// clang-format off
+#if defined(AXOM_USE_OPENMP)
+  using omp_exec = axom::OMP_EXEC;
+#else
+  using omp_exec = seq_exec;
+#endif
+
+#if defined(AXOM_USE_CUDA)
+  constexpr int CUDA_BLK_SZ = 256;
+  using cuda_exec = axom::CUDA_EXEC<CUDA_BLK_SZ>;
+#else
+  using cuda_exec = seq_exec;
+#endif
+
+// Hip exec included from IntersectionShaper header
+
+// clang-format on
+
+//-----------------------------------------------------------------------------
+/// Basic RAII utility class for initializing and finalizing slic logger
+//-----------------------------------------------------------------------------
+struct BasicLogger
+{
+  BasicLogger()
+  {
+    namespace slic = axom::slic;
+
+    // Initialize the SLIC logger
+    slic::initialize();
+    slic::setLoggingMsgLevel(slic::message::Debug);
+
+    // Customize logging levels and formatting
+    const std::string slicFormatStr = "[<LEVEL>] <MESSAGE> \n";
+
+    slic::addStreamToMsgLevel(new slic::GenericOutputStream(&std::cerr),
+                              slic::message::Error);
+    slic::addStreamToMsgLevel(
+      new slic::GenericOutputStream(&std::cerr, slicFormatStr),
+      slic::message::Warning);
+
+    auto* compactStream =
+      new slic::GenericOutputStream(&std::cout, slicFormatStr);
+    slic::addStreamToMsgLevel(compactStream, slic::message::Info);
+    slic::addStreamToMsgLevel(compactStream, slic::message::Debug);
+  }
+
+  ~BasicLogger() { axom::slic::finalize(); }
+};
+
+//-----------------------------------------------------------------------------
+/// Struct to help with parsing and storing command line args
+//-----------------------------------------------------------------------------
+enum class RuntimePolicy
+{
+  raja_seq = 1,
+  raja_omp = 2,
+  raja_cuda = 3,
+  raja_hip = 4
+};
```

Comment:
Please modify this example to use axom's runtime policies  (added here: https://github.com/LLNL/axom/pull/1225)